crates/forge/src/running.rs

```diff
@@ -151,18 +151,12 @@ fn build_syscall_handler<'a>(
         entry_point,
         string_to_hint,
         ReadOnlySegments::default(),
-    );
-    Ok(syscall_handler)
+    )
 }
 
-#[derive(Debug, Error)]
-pub(crate) enum TestCaseRunError {
-    #[error("{0}")]
-    RunnerError(#[from] RunnerError),
-    #[error("{0}")]
-    StateError(#[from] StateError),
-    #[error("{0}")]
-    Error(#[from] anyhow::Error),
+pub(crate) struct RunResultWithInfo {
+    pub(crate) run_result: Result<RunResult, RunnerError>,
+    pub(crate) block_number_of_latest: Option<BlockNumber>,
```

Comment:
```suggestion
    pub(crate) latest_block_number: Option<BlockNumber>,
```
I know that this isn't exactly the same semantically, but wouldn't it make it more readable?